src/components/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+
+import {useUniqId} from '../../../hooks';
+import {Disclosure} from '../../Disclosure';
+import {DefaultDisclosureSummary} from '../../Disclosure/DisclosureSummary/DisclosureSummary';
+import type {QAProps} from '../../types';
+import {useAccordionAttributes, useAccordionItems} from '../AccordionContext';
+import {AccordionSummary, isAccordionSummaryComponent} from '../AccordionSummary/AccordionSummary';
+import {
+    accordionDetailsBlock,
+    accordionItemBlock,
+    accordionSummaryTriggerBlock,
+} from '../constants';
+
+import './AccordionItem.scss';
+
+export type AccordionItemProps = Partial<{
+    value: string;
+    children: React.ReactNode;
+    expanded: boolean;
+    disabled: boolean;
+    keepMounted: boolean;
+    onUpdate: (expanded: boolean) => void;
+    defaultExpanded: boolean;
+    className: string;
+    summary: React.ReactNode;
+}> &
+    QAProps;
+
+export const AccordionItem = React.forwardRef<HTMLDivElement, AccordionItemProps>(
+    function AccordionItem(props, _ref) {
+        const {
+            children,
+            expanded,
+            defaultExpanded,
+            disabled,
+            keepMounted,
+            value,
+            summary,
+            onUpdate,
+            qa,
+        } = props;
+
+        const attributes = useAccordionAttributes();
+        const {id, isExpanded, handleUpdate} = useAccordionItemState({
+            expanded,
+            defaultExpanded,
+            value,
+            onUpdate,
+        });
+
+        const [preparedSummary, details] = React.useMemo(() => {
+            return prepareChildren(children, qa);
+        }, [children, qa]);
+
+        return (
+            <Disclosure
+                key={value ?? id}
+                arrowPosition={attributes?.arrowPosition}
+                keepMounted={keepMounted}
+                onUpdate={handleUpdate}
+                expanded={defaultExpanded ? undefined : isExpanded}
+                defaultExpanded={defaultExpanded}
+                summary={summary}
+                disabled={disabled}
+                qa={qa}
+                className={accordionItemBlock({
+                    size: attributes?.size,
+                    view: attributes?.view,
+                    disabled,
+                })}
+            >
+                {preparedSummary}
+                <Disclosure.Details className={accordionDetailsBlock}>{details}</Disclosure.Details>
+            </Disclosure>
+        );
+    },
+);
+
+function useAccordionItemState({
+    expanded,
+    defaultExpanded,
+    value,
+    onUpdate,
+}: {
+    expanded?: boolean;
+    defaultExpanded?: boolean;
+    value?: string;
+    onUpdate?: (next: boolean) => void;
+}) {
+    const id = useUniqId();
+    const {items, updateItems} = useAccordionItems();
```

Comment:
Using `useAccordionItems` makes it harder to read the code:
- `items` are only needed to define the `IsExpanded`, this can be done outside the `AccordionItem`
- it is better to define `updateItems` either in the general context or in the `Accordion`